main/src/main/res/values/strings.xml

```diff
@@ -7,79 +7,79 @@
     <string name="address">Server Address:</string>
     <string name="port">Server Port:</string>
     <string name="location">Location</string>
-    <string name="cant_read_folder">Unable to read directory</string>
+    <string name="cant_read_folder">Could not read directory</string>
```

Comment:
Not sure that is needed to change.

Reply:
The idea is to have consistency for every similar error message.
Then they can all be grouped as such in Weblate, making it easier to translate them all.
Arguably it is also a consistent experience between one error message to the next, which is a more
gracious fail.